content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
+
+### Configure a Slack automation
+Configuring a Slack integration takes multiple steps. First, [add one or more Slack channels]({{< relref "#add-a-slack-channel" >}}) as alert destinations. Next, [create the automation that notifies the Slack channel]({{< relref "#create-slack-automation" >}}).
+
+#### Add a Slack channel
+
+1. Go to Team Settings page.
+1. To integrate with a new Slack workspace and channel, click **Connect Slack**.
+    {{% alert %}}
+    To integrate with a new Slack channel in a workspace that is already set up, click **New integration**.
+    {{% /alert %}}
+
+  If necessary, sign in to your Slack instance. A browser window appears, asking that you grant Weights and Biases permission to post to the Slack channel you select. Read the page, then click **Search for a channel** and begin typing the channel name. Select the channel from the list, then click **Allow**.
+
+1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: Weights & Biases`, the integration is configured correctly.
+
+Now you can [create an automation that uses the Slack integration]({{< relref "#create-slack-automation" >}}).
+
+#### Create the automation {#create-slack-automation}
+
+After you [configure a Slack integration]({{< relref "#configure-the-slack-integration" >}}), follow these steps to create an automation that uses it.
+
+1. Go to the project page.
+1. In the sidebar, click **Automations**.
+1. Click **Create automation**.
+1. Choose the **Event** which triggers the automation. If applicable, provide options that are specific to the event type. If your project has no registries, registry events will not be available. Click **Next step**.
+1. Select the team where you added the Slack integration.
+1. Set **Action type** to **Slack notification**. Select the Slack channel, then click **Next step**.
+1. Proviude a name for the automation. Optionally, provide a description.
+1. Click **Create automation**.
+
+### Configure a webhook automation
+
+Configuring a webhook integration takes multiple steps:
+
+1. If your webhook requires any sensitive strings, [add them as secrets]({{< relref "#add-a-secret" >}}) for any sensitive strings required by the webhook. If the webhook requires a bearer token, do not configure a secret for it. Instead, you configure it as part of creating the webhook.
+1. [Create the webook]({{< relref "#add-a-webhook" >}}). If necessary, specify its access token, and grant it access to any secrets it needs.
+1. [Create an automation that uses the webhook]({{< relref "#create-webhook-automation" >}}), configuring the payload it sends to the third-party service.
+
+#### Add a secret
+A secret is a team-level variable that lets you obfuscate a sensitive string such as a credential, API key, password, or token. W&B recommends you use secrets to store any string that you want to protect the plain text content of.
```

Comment:
I know you're just copying the earlier text, but I would definitely avoid using the term `obfuscate` here - in security settings, it implies that we are not storing the secret in a secure manner. 

an AI summary I found describes them as "a piece of sensitive information, like a password, API key, or certificate, that is securely stored and managed within the Secret Manager service, allowing applications to access this data without exposing it directly within the code"

emphasizing that they are securely stored and then that you can use them without exposing them everywhere (eg in the automation config) are the main points to hit. (we should verify that we actually secure store them :)

Reply:
Not yet done.

Reply:
Done, i think!